Tests/SentryTests/Integrations/SentryFramesTrackerTests.swift

```diff
@@ -0,0 +1,131 @@
+import XCTest
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+class SentryFramesTrackerTests: XCTestCase {
+    
+    private class Fixture {
+        
+        var displayLinkWrapper: TestDiplayLinkWrapper
+        var queue: DispatchQueue
+        
+        init() {
+            displayLinkWrapper = TestDiplayLinkWrapper()
+            queue = DispatchQueue(label: "SentryFramesTrackerTests", qos: .background, attributes: [.concurrent])
+        }
+        
+        var sut: SentryFramesTracker {
+            return SentryFramesTracker(displayLinkWrapper: displayLinkWrapper)
+        }
+    }
+    
+    private var fixture: Fixture!
+    
+    override func setUp() {
+        fixture = Fixture()
+    }
```

Comment:
M: in theory, this is never needed unless you are testing static fields in the fixture too, what really makes sense is that every test creates a new `sut`, but not really the `Fixture`

Reply:
Hmm then maybe I use the fixture pattern a bit wrong. The problem I have quite often that the fixture contains properties needed for creating the sut that I also need for assertions. These properties have state. This state needs to be fresh for every test. Where would you put these properties then?

Reply:
I guess your approach would make more sense if using fakes instead of mocks, that's why they have state.
JUnit/mockito already resets everything so it's a bit easier, maybe your approach is the correct way when doing so.
good point